common/reflect_resource.go

```diff
@@ -254,6 +254,25 @@ func typeToSchema(v reflect.Value, t reflect.Type, path []string) map[string]*sc
 			scm[fieldName].Type = schema.TypeList
```

Comment:
Note to reviewers: disregard all changes in this file since it fixes some tests in the underlying architecture and will be merged in separately.